sdk/search/azure-search/pom.xml

```diff
@@ -28,13 +28,14 @@
   <dependencies>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-core</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.3.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-netty;current} -->
     </dependency>
     <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-core-http-netty</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Why is this adding Jackson as a compile time dependency? Azure Core has this as a transitive dependency which should be pulled in.

Reply:
The tests failed because of missing the required module. You can check the previous commit.

Reply:
Double check the modules dependencies. Remove all unnecessary ones.

Reply:
Revert the azure-core back as the transitive cannot apply more than multiple layers.